Formula/isync.rb

```diff
@@ -1,15 +1,11 @@
 class Isync < Formula
   desc "Synchronize a maildir with an IMAP server"
   homepage "https://isync.sourceforge.io/"
-  revision 1
   head "https://git.code.sf.net/p/isync/isync.git"
 
   stable do
```

Comment:
No need to keep the stable block without the patch. This can go into the class again